### PR TITLE
update master/worker API docs

### PR DESCRIPTION
**Pull request by rutsky**
Fix `remote_getCommands()` result.
Add missing fields of `remote_getWorkerInfo()` result.

### CODE REVIEW

**Comment by rutsky** [comment]
Built docs: https://circle-artifacts.com/gh/rutsky/buildbot/215/artifacts/0/home/ubuntu/buildbot/master/docs/_build/html/developer/master-worker.html#bot-methods

**Comment by tardyp** [comment]
:+1:

File: master/docs/developer/master-worker.rst [hunk 1]
```diff
@@ -39,7 +39,8 @@ Bot methods
 The worker-side Bot object has the following remote methods:
 
 :meth:`~buildbot_worker.pb.BotPb.remote_getCommands`
-    Returns a list of ``(name, version)`` for all commands the worker recognizes.
+    Returns a dictionary ``Dict[name, version]`` for all commands the worker recognizes.
```
Comment (djmitche) [inline master/docs/developer/master-worker.rst]: I'm not sure what this notation means -- are names the keys and versions the values?

**Comment by djmitche** [comment]
Please merge with that notation clarified.

File: master/docs/developer/master-worker.rst (hunk 1 above)
Comment (rutsky) [inline master/docs/developer/master-worker.rst]: What was meant here is that the result is Python `dict` with names as the keys, and versions as the values, yes.
I used notation of [PEP 484](https://www.python.org/dev/peps/pep-0484/) which might be used with `typing` module in Python 3.

I'll replace this with explanation in English.

**Comment by rutsky** [comment]
@djmitche I rephrased that part removing PEP 484 notation.

**Comment by djmitche** [comment]
assuming tests pass, this is good to go

**Comment by djmitche** [comment]
```
/home/travis/build/buildbot/buildbot/master/docs/developer/plugins-publish.rst:59: WARNING: broken link: http://pythonhosted.org/setuptools/setuptools.html#dynamic-discovery-of-services-and-plugins
```

**Comment by rutsky** [comment]
Huh, and it's 404 for me. Let me find fix for this one...

**Comment by rutsky** [comment]
Done!

**Comment by djmitche** [comment]
OK thanks -- go ahead and merge when the tests come back green

**Pull request merged**